### ISSUE TITLE
sc output of options used

### ISSUE THREAD

**Issue by donnaaboise** [ForestClaw/forestclaw#103]
Originally reported by: **Donna Calhoun (Bitbucket: [donnaaboise](https://bitbucket.org/donnaaboise), GitHub: [donnaaboise](https://github.com/donnaaboise))**

----------------------------------------

every once in a while, the [fclaw] and [p4est] options printing at the beginning of a run get interspersed, making it hard to parse the file.  

```
[fclaw]    noweightedp            false
[p4est 58] Local minimum/maximum levels:  4  4
[fclaw]    trapfpe                false
[p4est 59] Patches on level  4:         4
[fclaw]    mpi_debug              false
[p4est 58] Patches on level  4:         4
[fclaw]    ghost_patch_pack_area  true
[p4est 55] Local minimum/maximum levels:  4  4
[fclaw]    mi                     1
[p4est 54] Local minimum/maximum levels:  4  4
[fclaw]    mj                     1
[p4est 55] Patches on level  4:         4
[fclaw]    periodic_x             true
[p4est 54] Patches on level  4:         4
[fclaw]    periodic_y             true
[p4est 57] Local minimum/maximum levels:  4  4
```

Can a barrier, or some other feature be added so that this doesn't happen?

This has only happened twice now, and so it is easy enough to edit the output file.  But the less manual intervention that is needed, but better. 

----------------------------------------
- Bitbucket: https://bitbucket.org/donnaaboise/forestclaw/issue/103

**Comment by cburstedde** [comment]
A barrier might help, but it's no guarantee.
Production runs should have no output from any other procs than 0 anyway -- is this observed?

**Comment by donnaaboise** [comment]
I should develop a better way to parse output files - probably reading a summary line is the best way to get.  Alternatively, it is straightforward to write out a binary file with all of the output that can be easily read by Python.

**Comment by cburstedde** [comment]
Have we activated the summary line in sc_stats_print? It gives you the averages and maxima and is easy to parse (matlab-style).
